dist/src/main/java/io/zeebe/EnvironmentHelper.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe;
+
+import java.util.Arrays;
+import java.util.List;
+import org.springframework.core.env.Environment;
+
+public class EnvironmentHelper {
+
+  public static boolean isProductionEnvironment(Environment springEnvironment) {
+    boolean result = true;
+
+    if (springEnvironment == null) {
+      result = false;
+    } else {
+      final String[] activeProfiles = springEnvironment.getActiveProfiles();
+      if (activeProfiles != null && !(activeProfiles.length == 0)) {
```

Comment:
I think it's not possible for `activeProfiles` to be null according to their source code - they return a static empty array instance. Although I guess since they don't annotate it as `NonNull` it could change? What do you think?

Reply:
I think you are right. It's mostly an old habit of mine to use defensive programming. 

Also, I wouldn't want to use many assumptions about the implementation of code I use. Plus it's an interface.

Again, in a test, if you pass in a mock for the `springEnvironment` which isn't set up completely, it will return null.

All in all, I think you are correct. However, I would like to keep it that way,

Reply:
No, you're right, let's go with that.